### ISSUE TITLE
[NEW BUILD - DEFAULT / 19w32a] Hotfixes!

### ISSUE THREAD

**Issue by tsunamayo** [tsunamayo/Skywanderers#1408]
Quick fix for the texture issue of 19w31b. Thanks!

**Comment by AfterRebelion** [comment]
Maybe "fix" is not what I would call it...
![imagen](https://user-images.githubusercontent.com/559819/62465153-8b311700-b78e-11e9-8ebc-f594207d0a52.png)
![imagen](https://user-images.githubusercontent.com/559819/62465171-91bf8e80-b78e-11e9-944d-033e1281f103.png)

**Comment by tsunamayo** [comment]
I started working on the new Damage system, so that might be related to that...
The new tech will indeed cut through your ship where voxel are destroyed.
Did you do something special? (it is fine on my side)
Cheers

**Comment by tsunamayo** [comment]
Okay I just noticed there was also another major bug on the building side, so I am taking this build down and will do a cleaner update once everything is fixed! I keep you posted.

**Comment by AfterRebelion** [comment]
> Did you do something special? (it is fine on my side)
> Cheers
Nothing special, just loaded the game, and this happened. One thing to take note is that it happens on somewhat long freeformed blocks (Like 10 meters long)

**Comment by tsunamayo** [comment]
Yes actually on one of my other save I got this bug, so I will be able to fix it. I keep you posted, thanks!
